fuzzers/041-clk-hrow-pips/generate.py

```diff
@@ -2,9 +2,16 @@
 
 import os
 import os.path
+import re
 from prjxray.segmaker import Segmaker
 
 
+def is_fake_feature(src):
```

Comment:
"is_fake_feature" is a poor name.  Maybe "src_has_active_bit" and invert the boolean output

Reply:
This does sound like a better name. Done.